cypress/integration/features/task/createEditAndDeleteTask.feature

```diff
@@ -20,13 +20,15 @@ Feature: To create, edit and delete tasks by the teacher.
     Then I can see room page 'Course with subject and tasks'
     And I can see task 'Cy Task Creating and Deleting Test'
 
-Scenario: Teacher edits and publishes task from room via form
+  Scenario: Teacher edits and publishes task from room via form
     Given I am logged in as a 'teacher1' at 'brb'
     When I go to rooms overview
     And I go to room 'Course with subject and tasks'
     When I click on three dot menu of content 'Cy Task Creating and Deleting Test'
     And I click on Edit in dot menu
-    # Then file upload button is enabled
+    Then file upload button is enabled
```

Comment:
can be-> Then I see file upload button is enabled

Reply:
done.